content/posts/2025-plans.md

```diff
@@ -1,25 +1,28 @@
 ---
 title: "Open Ecoacoustics 2025 Plans"
-date: 2025-02-05T12:19:00+10:00
-draft: true
+date: 2025-02-06T12:19:00+10:00
+image: '/images/sensor-occupant.png'
 tags:
 - News
 ---
 I hope your 2025 is off to a great start. We at Open Ecoacoustics are excited to keep the community more updated on our activities and progress. This year, we're committed to providing more regular updates and sharing our work through these posts and on social media. 
-
 <!--more-->
+
 ## Follow Us on BlueSky and LinkedIn 
 
 We are excited to announce the launch of our accounts on [BlueSky](https://bsky.app/profile/openecoacoustics.bsky.social) and [LinkedIn](https://www.linkedin.com/company/open-ecoacoustics/). Follow us to stay in the loop! 
 
+_Cover image of this post and our social media profiles by Prof. Susan Fuller_
+
+## New Feature: Audio Verification Interface
+Last year we finished the first version of an audio verification component. This is an open-source component that can be embedded on any website. We are currently working on adding it to [Ecosounds](https://www.ecosounds.org/). Here is a quick gif to show you how it works:
+{{% figure src="images/verification-interface.gif" caption="Verification Interface." width="50%"%}}
```

Comment:
Let avoid talking about the reusability - we can talk about that in the web components post.

Can we explain what the verification interface is for? When it will be available? What features we're missing. Let' set some expectations here and actually describe what it does.